### ISSUE TITLE
[1.4] Issue with bcrypt

### ISSUE THREAD

**Issue by dnish** [andruschka/pm2-meteor#23]
Well,
I don't know if this is an issue with Meteor with pm2-meteor, but when deploying my Meteor app to a Ubuntu 16.04 server, I get the following error on startup:

```
app-0 Error: Module did not self-register.
app-0     at Error (native)
app-0     at Object.Module._extensions..node (module.js:434:18)
app-0     at Module.load (module.js:343:32)
app-0     at Module.Mp.load (/opt/pm2-meteor/app/app/bundle/programs/server/npm/node_modules/meteor/babel-compiler/node_modules/reify/node/runtime.js:16:23)
app-0     at Function.Module._load (module.js:300:12)
app-0     at Function._load (/usr/lib/node_modules/pm2/node_modules/pmx/lib/transaction.js:62:21)
app-0     at Module.require (module.js:353:17)
app-0     at require (internal/module.js:12:17)
app-0     at bindings (/opt/pm2-meteor/app/app/bundle/programs/server/npm/node_modules/meteor/npm-bcrypt/node_modules/bindings/bindings.js:76:44)
app-0     at Object.<anonymous> (/opt/pm2-meteor/app/app/bundle/programs/server/npm/node_modules/meteor/npm-bcrypt/node_modules/bcrypt/bcrypt.js:3:35)
```

Removing bcrypt from the server folder and installing it again via npm solves the problem. I've set the "--architecture os.linux.x86_64" flag in my config.

**Comment by dnish** [comment]
@andruschka Maybe we should add some script that is executed after uploading and unzipping the bundle, to automatically install the bcrypt dep on the host system. See issue https://github.com/meteor/meteor/issues/7512

**Comment by zeroasterisk** [comment]
@dnish what commands did you run to uninstall and re-install bcrypt?

**Comment by dnish** [comment]
@zeroasterisk This were the steps I had to do:

```
rm -r /opt/pm2-meteor/app/app/bundle/programs/server/npm/node_modules/meteor/npm-bcrypt/node_modules/bcrypt

cd /opt/pm2-meteor/app/app/bundle/programs/server

npm install bcrypt
```

But from the official Meteor issue, this may also work:

```
cd  <root app folder>/bundle/programs/server/npm/node_modules/meteor/npm-bcrypt/node_modules/bcrypt
node-gyp rebuild
```

You can reproduce this with every 1.4 application that uses the accounts package.

**Comment by dnish** [comment]
Seems to be fixed with Meteor 1.4.0.1
